projects/ad9656/common/ad9656_qsys.tcl

```diff
@@ -0,0 +1,99 @@
+# ad9656_rx JESD204
+
+add_instance ad9656_rx_jesd204 adi_jesd204
+set_instance_parameter_value ad9656_rx_jesd204 {ID} {1}
+set_instance_parameter_value ad9656_rx_jesd204 {TX_OR_RX_N} {0}
+set_instance_parameter_value ad9656_rx_jesd204 {SOFT_PCS} {true}
+set_instance_parameter_value ad9656_rx_jesd204 {LANE_RATE} {9830.4}
```

Comment:
If the Intel support is added, please make sure parameters are correct. I would not add Intel support at this time though.

Reply:
Intel support was not added. Ad9656_qsys.tcl file deleted from the pull request.